.github/workflows/release.yaml

```diff
@@ -78,7 +78,6 @@ jobs:
             VERSION=${{ env.VERSION }}
 
       - name: prepare-cli-manifests # this step will ensure that kusk kustomize manifests are build with the latest image tag github.sha
-        uses: actions/setup-go@v2
         with:
```

Comment:
This isn't going to work. We need to set up go with 1.19

Put `VERSION=${{ env.VERSION }} make manifests` into a separate step